com.gitlab.adnan338.Invoicer.yaml

```diff
@@ -20,13 +20,13 @@ modules:
       - "*"
     build-commands:
       - >
-          if [ $(uname -m) == "x86_64" ]; then 
+          if [ $(uname -m) == "x86_64" ]; then
```

Comment:
you could also change this (and below) to have `only-arches`. This would save a lot of downloads.

Reply:
Thanks, done.